wgpu-core/src/id.rs

```diff
@@ -154,13 +154,9 @@ where
     T: 'static + WasmNotSendSync,
 {
     fn fmt(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
-        let (index, epoch, backend) = self.unzip();
-        formatter
-            .debug_struct("Id")
-            .field("index", &index)
-            .field("epoch", &epoch)
-            .field("backend", &backend)
-            .finish()
+        let (index, epoch, _) = self.unzip();
+        write!(formatter, "Id({index},{epoch})")?;
```

Comment:
Could we keep the backend in here (maybe a shortened version of it [vk, 12, 11, mtl, gl])

Reply:
Alright, done!